app/uk/gov/hmrc/valuationofficeagencycontactfrontend/views/govuk/main_template.scala.html

```diff
@@ -0,0 +1,62 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.layouts.govukLayout
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.helpers._
+
+@import uk.gov.hmrc.valuationofficeagencycontactfrontend.views.html.govuk.head
+@import uk.gov.hmrc.valuationofficeagencycontactfrontend.views.html.govuk.scripts
+
+
+@this(
+        govukLayout: govukLayout,
+        head: head,
+        scripts: scripts,
+        govukPhaseBanner: govukPhaseBanner
+)
+@(pageTitle: Option[String] = None,
+        beforeContentBlock: Option[Html] = None
+)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
+
+@beforeBLock = {
+    @govukPhaseBanner(PhaseBanner(tag = Option(Tag(Text("BETA"))),
+        content = HtmlContent("This is a new service – your <a class=\"govuk-link\" href=\"#\" target=\"_blank\" id=\"betaFeedback\">feedback</a> will help us to improve it.")))
+    @beforeContentBlock
+
+}
+
+
+@govukLayout(
+    pageTitle = pageTitle,
+    headBlock = Some(head()),
+    beforeContentBlock = Option(beforeBLock),
+    footerItems = Seq(
+        FooterItem(
+            href = Some(messages("footer.links.cookies.text")),
+            text = Some(messages("footer.links.cookies.text"))),
+        FooterItem(
```

Comment:
Did you mean both cookie messages to be the same? Should one of the be .url?

The same question holds for privacy policy below.